serverless.yaml

```
@@ -167,6 +172,39 @@ resources:
               - Key: 'service'
                 Value: 'fhir'
             - Ref: AWS::NoValue
+      ExportRequestDynamoDBTable:
+        Type: AWS::DynamoDB::Table
+        Properties:
+          AttributeDefinitions:
+            - AttributeName: jobId
+              AttributeType: S
+            - AttributeName: jobStatus
+              AttributeType: S
+            - AttributeName: jobOwnerId
+              AttributeType: S
+          KeySchema:
+            - AttributeName: jobId
+              KeyType: HASH
+          TableName: '${self:custom.exportRequestTableName}'
+          SSESpecification:
+            KMSMasterKeyId: !Ref DynamodbKMSKey
+            SSEEnabled: true
+            SSEType: 'KMS'
+          BillingMode: PAY_PER_REQUEST
+          GlobalSecondaryIndexes:
+            - IndexName: '${self:custom.exportRequestTableJobStatusIndex}'
+              KeySchema:
+                - AttributeName: jobStatus
+                  KeyType: HASH
+                - AttributeName: jobOwnerId
+                  KeyType: RANGE

```

Not sure if this matters but I assume `ownerId` will be more unique than `jobStatus` could we flip which is primary & sort key?
```suggestion
                - AttributeName: jobOwnerId
                  KeyType: HASH
                - AttributeName: jobStatus
                  KeyType: RANGE

```

---

Ah the reason for `jobStatus` being the hashKey of the index is so we can query on `jobStatus` by itself. I think if `jobStatus` was the rangeKey, we woudn't be able to search for jobs that has `jobStatus === in-progress`